server/src/test/java/io/crate/expression/reference/sys/snapshot/SysSnapshotsTest.java

```diff
@@ -113,14 +123,65 @@ public void testUnavailableSnapshotsAreFilteredOut() throws Exception {
 
     @Test
     public void test_current_snapshot_does_not_fail_if_get_repository_data_returns_failed_future() throws Exception {
-        Repository r1 = mock(Repository.class);
+        Repository r1 = mock(Repository.class, Answers.RETURNS_DEEP_STUBS);
```

Comment:
why do you need the RETURNS_DEEP_STUBS here?

Reply:
because chan of calls is used (r1.**getMetadata().name()**). Otherwise would have to mock the whole chain.

Reply:
thx!